cheatsheets/XML_External_Entity_Prevention_Cheat_Sheet.md

```diff
@@ -549,6 +549,24 @@ libxml_disable_entity_loader(true);
 
 A description of how to abuse this in PHP is presented in a good [SensePost article](https://www.sensepost.com/blog/2014/revisting-xxe-and-abusing-protocols/) describing a cool PHP based XXE vulnerability that was fixed in Facebook.
 
+# PYTHON
+
+The following information for XXE injection in PYTHON 3 is directly from this [official PYTHON 3 documentation page](https://docs.python.org/3/library/xml.html#xml-vulnerabilities).
+
+The below table gives an overview of various modules in PYTHON 3 used for XML parsing and whether they are vulnerable or not.
+
+| Attack Type               | sax        | etree      | minidom    | pulldom    | xmlrpc     |
+|---------------------------|------------|------------|------------|------------|------------|
+| Billion Laughs            | Vulnerable | Vulnerable | Vulnerable | Vulnerable | Vulnerable |
+| Quadratic Blowup          | Vulnerable | Vulnerable | Vulnerable | Vulnerable | Vulnerable |
+| External Entity Expansion | Safe       | Safe       | Safe       | Safe       | Safe       |
+| DTD Retrieval             | Safe       | Safe       | Safe       | Safe       | Safe       |
+| Decompression Bomb        | Safe       | Safe       | Safe       | Safe       | Vulnerable |
+
+For details on PYTHON 2, refer this [official PYTHON 2 documentation page](https://docs.python.org/2/library/xml.html#xml-vulnerabilities). Kindly note that PYTHON 2 End of Life has been announced for January 1st 2020. After that PYTHON 2 will become obsolete and no official support will be available.
```

Comment:
`Kindly` is an odd word (I can't remember which English variant uses it, but it isn't standard for en-US). I'd suggest `Do` instead, or simply omit the word entirely (as in `Note:`).

Reply:
Note: it's more common to write `the end of life for X` or `X has reached end of life` as opposed to `X end of life` (this isn't about the casing `End of Life`/`end of life`, just the phrasing).

For Python, it hasn't technically reached end of life, but the same general idea applies, as in:
`the end of life for X will/is-expected-to be ...` or `X will reach end of life...`.